app/services/statements/show_service.rb

```diff
@@ -0,0 +1,7 @@
+module Statements
+  class ShowService
```

Comment:
I think we do because it might just do `Statement.all` today but tomorrow it might log, filter, sort, split into multiple smaller lists, and this provides a natural place to put that.